Description: Debianize the version numbers of maven artefacts
 in the antrun target for the web application. This is not 
 transformed by maven-debian-helper.
Author: James Page <james.page@ubuntu.com>
Forwarded: not-needed

Index: jenkins/war/pom.xml
===================================================================
--- jenkins.orig/war/pom.xml	2011-11-22 08:20:18.471237994 +0000
+++ jenkins/war/pom.xml	2011-11-22 08:20:32.891226274 +0000
@@ -109,7 +109,7 @@
                 <mkdir dir="target/generated-resources" />
 
                 <!-- put executable war header -->
-                <resolveArtifact groupId="org.jenkins-ci" artifactId="executable-war" version="1.19" type="jar" property="executable-war.jar" />
+                <resolveArtifact groupId="org.jenkins-ci" artifactId="executable-war" version="debian" type="jar" property="executable-war.jar" />
                 <unjar src="${executable-war.jar}" dest="target/generated-resources">
                   <patternset>
                     <include name="**/*.class" />
